src/features/environmentCreate/components/EnvironmentCreate.tsx

```diff
@@ -95,9 +95,9 @@ export const EnvironmentCreate = ({ environmentNotification }: IEnvCreate) => {
           description: createLabel(name, "create")
         }
       });
-    } catch ({ data }) {
+    } catch (e) {
       setError({
-        message: data.message,
+        message: e?.data?.message ?? createLabel(undefined, "error"),
```

Comment:
This used to fail because `data` was `undefined`.

A repro: try to create an environment with 256 `a` chars as the env name: `'a' * 256`. This causes a SQL error on the server because the field is only 255 chars long. With this change, the UI now shows an error message.

I'm not adding a "before" screenshot because before it was just showing nothing - same page you have when you click Create. You could also see an uncaught exception in the dev browser console (that's usually opened by pressing f12).

<img width="796" alt="Screen Shot 2023-11-08 at 21 06 21" src="https://github.com/conda-incubator/conda-store-ui/assets/10469885/b8ed8c82-c5f5-4628-81d4-b010b49a5ac8">

Reply:
What are the possible error messages that will be displayed here?

Reply:
`e?.data?.message` corresponds to what used to be here before. I think `message` is not actually used by the said route, but I would keep it just in case in case it's changed later. The route on the server is:
```
api_post_specification
...
return {"status": "ok", "data": {"build_id": build_id}}
```
Unlike some other routes, it doesn't include the `message` field.

The new message is always a generic error message shown on the screenshot above. This is what we do elsewhere as well, so it's consistent.